nucliadb/src/nucliadb/search/search/rank_fusion.py

```diff
@@ -141,21 +152,27 @@ def __init__(
         self._k = k
         self._keyword_boost = keyword_weight
         self._semantic_boost = semantic_weight
+        self._graph_boost = graph_weight
 
     @rank_fusion_observer.wrap({"type": "reciprocal_rank_fusion"})
     def _fuse(
-        self, keyword: Iterable[TextBlockMatch], semantic: Iterable[TextBlockMatch]
+        self,
+        keyword: Iterable[TextBlockMatch],
+        semantic: Iterable[TextBlockMatch],
+        graph: Iterable[TextBlockMatch],
```

Comment:
At this point, maybe this should take an array of things to merge? Something like `list[tuple[Iterable[TextBlockMatch], float]]`? Just make this algorithm generic and when calling it, pass what's needed (no need to pass an empty list)

Reply:
For the difference it makes, I'd leave it like that. Otherwise, you'll need to know the boosting at call time, and it'd be a specific parameter for RRF and, even if it's the only one we have, I don't think it's worth it